quantecon/util/numba.py

```diff
@@ -0,0 +1,71 @@
+"""
+Utilities to support Numba jitted functions
+
+"""
+import numpy as np
+from numba import generated_jit, types
+from numba.targets.linalg import _LAPACK
+
+
+# BLAS kinds as letters
+_blas_kinds = {
+    types.float32: 's',
+    types.float64: 'd',
+    types.complex64: 'c',
+    types.complex128: 'z',
+}
+
+
+@generated_jit(nopython=True, cache=True)
+def _numba_linalg_solve(a, b):
+    """
+    Solve the linear equation ax = b directly calling a Numba internal
+    function. The data in `a` and `b` are interpreted in Fortran order,
+    and dtype of `a` and `b` must be the same, one of {float32, float64,
+    complex64, complex128}. `a` and `b` are modified in place, and the
+    solution is stored in `b`. *No error check is made for the inputs.*
```

Comment:
I am not sure if I understand your question, but [here](https://github.com/QuantEcon/QuantEcon.py/blob/20da0f8872fae93d5adc4c5e3b95582879263ba3/quantecon/game_theory/support_enumeration.py#L161-L163) is where this is used, where it is check whether matrix `A` is nonsingular (instead of `try`-`except` which is not available in nopython mode), and in case `A` is singular I don't know what is in there in `b`.